### ISSUE TITLE
Boundary event attachment options lack specific titles

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#5080]
### Describe the bug

Each of the boundary event options for attachment is labeled with a generic "Attach boundary event" message.

https://github.com/user-attachments/assets/87257770-62e2-46f4-8ae2-6493e96525c4

### Steps to reproduce

1. Create a task
2. Hover attach options
3. now this happens

### Expected behavior

Titles should be event-specific, e.g. _attach message boundary event_.

### Environment

- OS: [e.g. MacOS 10.2, Windows 10]
- Camunda Modeler Version: [e.g. 2.0.0]
- Execution Platform: [e.g. Camunda 7, Camunda 8]
- Installed plug-ins: [...]

### Additional context

Reported via slack: https://camunda.slack.com/archives/C0693F1NFK5/p1749247462508659

**Comment by barmac** [comment]
Fixed upstream via https://github.com/camunda/improved-canvas/pull/79
